scripts/airgapped/get-all-images.sh

```diff
@@ -46,6 +46,9 @@ rm -rf resource-dispatcher
 # manually retrieve pipelines runner image to test pipelines
 IMAGES+=($(echo "charmedkubeflow/pipelines-runner:ckf-1.8"))
 
+# manually retrieve helloworld image to test knative
+IMAGES+=($(echo "ghcr.io/knative/helloworld-go:latest"))
```

Comment:
Is this image tagged or is it always the latest tag?

Reply:
I see always `latest` being used even in the [docs repo release-1.10 branch](https://github.com/knative/docs/blob/release-1.10/docs/getting-started/first-service.md) and across all branches
the [image](https://github.com/knative/docs/tree/release-1.10/code-samples/serving/hello-world/helloworld-go) has not been touched for 3 years now